cmd/jaeger/internal/integration/integration.go

```diff
@@ -67,3 +71,41 @@ func (s *E2EStorageIntegration) e2eCleanUp(t *testing.T) {
 	require.NoError(t, s.SpanReader.(io.Closer).Close())
 	require.NoError(t, s.SpanWriter.(io.Closer).Close())
 }
+
+func createStorageCleanerConfig(t *testing.T, configFile string) string {
+	data, err := os.ReadFile(configFile)
+	require.NoError(t, err)
+	var config map[interface{}]interface{}
+	err = yaml.Unmarshal(data, &config)
+	require.NoError(t, err)
+
+	service, ok := config["service"].(map[interface{}]interface{})
+	require.True(t, ok)
+	service["extensions"] = []string{"jaeger_storage", "jaeger_query", "storage_cleaner"}
+
+	extensions, ok := config["extensions"].(map[interface{}]interface{})
+	require.True(t, ok)
+	trace_storage := getTraceStorage(extensions)
+	extensions["storage_cleaner"] = map[string]string{"trace_storage": trace_storage}
+
+	newData, err := yaml.Marshal(config)
+	require.NoError(t, err)
+	tempFile, err := os.Create("storageCleaner_config.yaml")
+	require.NoError(t, err)
+	err = os.WriteFile(tempFile.Name(), newData, 0o600)
```

Comment:
```suggestion
	tempFile := filepath.Join(t.TempDir(), "mapping.json")
	err = os.WriteFile(tempFile, newData, 0o600)
```
the dir will be automatically deleted, you don't need to rm it manually